src/test/java/io/vantiq/client/intg/VantiqIntegrationTest.java

```diff
@@ -915,6 +920,12 @@ public void testUploadAndDownloadImageHelper(String fileName, String contentType
         waitForCompletion();
 
         // Ensure the file uploaded successfully
+        if (handler.error) {
```

Comment:
if error & error due to VA server not installed, just skip the test.  Tester will re-run with proper environment.